src/main/java/fi/metatavu/metamind/weka/RecommendationData.java

```diff
@@ -0,0 +1,108 @@
+package fi.metatavu.metamind.weka;
+
+import java.util.ArrayList;
+
+import weka.core.Attribute;
+import weka.core.DenseInstance;
+import weka.core.Instance;
+import weka.core.Instances;
+
+/**
+ * Helper class for converting data
+ * 
+ * @author Simeon Platonov
+ */
+public class RecommendationData {
```

Comment:
Class naming is bit confusing, better would be e.q. RecommendationDataUtils

Reply:
Class naming changed